pkgs/applications/misc/coffeegrindsize/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, python3, fetchFromGitHub }:
+
+python3.pkgs.buildPythonApplication {
+  pname = "coffeegrindsize";
+  # no tags in the repo
+  version = "unstable-2021-04-20";
```

```suggestion
  version = "0-unstable-2021-04-20";
```

new guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

---

Thanks! I've missed that change